Tests/kaas/plugin/run_sonobuoy_executor.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
```

Comment:
I don't think the Sonobuoy stuff is relevant for this issue. What we want is to provision and unprovision clusters so they can be used for whatever scripts necessary. This script as well as `sonobuoy_executor.py` should be part of another PR.

Reply:
Removed it from this PR